roles/helm/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: install helm
+  become: yes
+  become_user: root
+  command: "curl -s {{ helm_url }} | bash"
+  args:
+    chdir: "{{ helm_folder }}"l
+    creates: "{{ helm_bin }}"
+  environment: "{{ proxy_env }}"
+  tags:
+    - helm
+
+- name: install helm packages
+  become: yes
+  become_user: root
+  run_once: True
+  when: "helm_{{ item.name }}_enabled | bool"
```

Comment:
mm we can ensure package is not installed when helm_{{ item.name }}_enabled=false
Not sure what you mean in the first bit, I like toggle thing so people can ensure the state if their cluster by overriding defaults with env variables.